### PR TITLE
Feature/add flat documentation3

### PR DESCRIPTION
**Pull request by stefanspiska**

### CODE REVIEW

File: source/09_flat/00_description/index.rst [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+  "conformance-ehrbase.de.v0/context/setting|value": "other care",
+  "conformance-ehrbase.de.v0/context/setting|terminology": "openehr",
+  "conformance-ehrbase.de.v0/conformance_section/conformance_observation/any_event:0/dv_quantity|magnitude": 65.9,
+  "conformance-ehrbase.de.v0/conformance_section/conformance_observation/any_event:0/dv_quantity|unit": "unit"
+  }
+
+
+Web-template 
+--------------
+The Flat format is based on a simplified representation of the template the so called Web-template.
```
Comment (birgerhaarbrandt) [inline source/09_flat/00_description/index.rst]: of a template, the so-called Web-Template

File: source/09_flat/00_description/index.rst [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+   }
+
+
+Flat Path
+--------------
+
+To build a Flat Path
+
+* add the id from the `Web-template`_ together
+* if a element is multi valued add a index
```
Comment (birgerhaarbrandt) [inline source/09_flat/00_description/index.rst]: an index

File: source/09_flat/00_description/index.rst [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+
+Flat Path
+--------------
+
+To build a Flat Path
+
+* add the id from the `Web-Template`_ together
+* if a element is multi valued add an index
+* Once at A Data Value uses "|" to select the Attribute
```
Comment (maximni-vg) [inline source/09_flat/00_description/index.rst]: spelling: once at a data value, use "|" to select the attribute

**Comment by wolandscat** [comment]
@birgerhaarbrandt alerted me to this change... 

Q1: I see things like conformance_section, conformance_observation etc - what does that mean? 

Q2: Also, I see .../any_event/DV_QUANTITY/... what is the DV_QUANTITY in this case?

**Comment by stefanspiska** [comment]
> @birgerhaarbrandt alerted me to this change...
> 
> Q1: I see things like conformance_section, conformance_observation etc - what does that mean?
> 
> Q2: Also, I see .../any_event/DV_QUANTITY/... what is the DV_QUANTITY in this case?

All example are bases on the template https://github.com/ehrbase/openEHR_SDK/blob/develop/test-data/src/main/resources/operationaltemplate/conformance_ehrbase.de.v0.opt

in which we named each rm object is named like its class

**Comment by stefanspiska** [comment]
Unbenannt

**Comment by wolandscat** [comment]
So in the above, a path like `.../any_event/DV_QUANTITY/` is really a shortened version of `.../events[id3]/data[id4]/items[id12]/value`. I would expect to see something like `.../any_event/items/weight`, assuming `id12` = weight. I don't think a path like `.../any_event/DV_QUANTITY/` is any use - it appears to be derived from the tool visualisation, which is not quite the same as the shape of the data hierarchy.

**Comment by stefanspiska** [comment]
@wolandscat 

see https://github.com/ehrbase/documentation/blob/feature/add_flat_documentation3/source/09_flat/00_description/index.rst

A Flat path is simply  the id of the webtemplate nodes  put together.

Where the Id of a Node in the webtemplate is the name of the opt node  (normalized and made uniqe)
where additional some node like (History and Element) form the opt  are left out from the webtemplate. 

That's the way it was defined by better and that the way the community is using it.

**Comment by wolandscat** [comment]
Well there we see paths like `conformance-ehrbase.de.v0/conformance_section/conformance_observation/any_event:0/dv_quantity|magnitude` - problem is we don't know what that means (could be weight, heart rate, or anything) - there can easily be more than one child of type DV_QUANTITY below any_event. So I can't see how this will work. I'll do some external checking...

Also, the idea for data types is the compact format - [see here](https://specifications.openehr.org/releases/SM/latest/serial_data_formats.html#_openehr_reference_model_data_value_types).

**Comment by wolandscat** [comment]
Ah - I see - it is some test template that has data type names for the leaf ELEMENTs. Very strange!

**Pull request merged**